### ISSUE TITLE
plugin not working on 1.37.0-insider

### ISSUE THREAD

**Issue by AnthonyParkssonos** [stef-levesque/vscode-perforce#167]
perforce.login command not found, can't do anything with the plugin, and debug output is not displaying.

64 bit ubuntu, using insiders edition of vscode.

i have a .p4config setup in home and in workspace directory, with the environment variable P4CONFIG exported, but cannot start perforce. 

i have also added the appropriate settings to my settings.json, and that has not resolved the issue after window reload.

any help is much appreciated!

**Comment by jdarnold** [comment]
Yeah, I have 1.38.0-insider on Windows and the plugin isn't working. The Perforce commands show up in the command palette, but if I try to run any of them, Code just says the command is "not found".

I've tried uinstalling, restarting Code, and re-installing. No luck

**Comment by navin09** [comment]
I am also facing this issue since I upgraded to version 1.38.0 running on Linux Mint 19.2 (Tina). It used to work fine on previous versions of VS Code. Funny thing is, it works *sometimes* but most times does absolutely nothing, like checking out a file using the shortcut Alt+P, E would not check the file out and I get no messages. I should add that I connect to my Perforce server using an SSH tunnel forwarding port 1666 to localhost. Using the graphical P4V client works perfectly. 

````
Version: 1.38.0
Commit: 3db7e09f3b61f915d03bbfa58e258d6eee843f35
Date: 2019-09-03T21:51:09.716Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Linux x64 4.15.0-58-generic
````

**Comment by mmakrzem** [comment]
I'm on 1.39.0-insider and this plugin is still broken.  Used to work fine.
